### DISCUSSION TITLE
How do I concatenate the eigenvalues of a node with the eigenvalues of the edges of that node

### DISCUSSION THREAD

**Discussion by yaoysyao** [pyg-team/pytorch_geometric#7385] [Q&A]
As a first-time user of pyg, I am not familiar with many relevant methods. I would like to know that my task is graph classification. When I use the training model of graph neural network, I want to integrate the eigenvalues of each graph node in batchsize with the edge of this node in the forward method. Is there an associated method for PYG to get the eigenvalue of the edge of a node?

**Comment by rusty1s** [comment]
[This](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/transforms/add_positional_encoding.py#L35) code-block might be useful for you.
